pkg/reconciler/kubernetes/tektonpipeline/transform.go

```diff
@@ -166,6 +170,85 @@ func updatePerformanceFlagsInDeployment(pipelineCR *v1alpha1.TektonPipeline) mf.
 	}
 }
 
+// updates resolver config environment variables
+func updateResolverConfigEnvironmentsInDeployment(pipelineCR *v1alpha1.TektonPipeline) mf.Transformer {
+	return func(u *unstructured.Unstructured) error {
+		if u.GetKind() != "Deployment" || u.GetName() != pipelinesControllerDeployment {
```

Comment:
this should be resolver

Reply:
my bad, I thought it goes into pipelines controller. I will update this.
Thanks @piyush-garg !